media-gfx/qview/qview-3.0.recipe

```diff
@@ -59,5 +59,20 @@ INSTALL()
 {
 	mkdir -p $appsDir
 	cp bin/qview $appsDir/qView
+
+	local APP_SIGNATURE="application/x-vnd.qview"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
```

Comment:
Because the recipe name has only major&middle version, "qview-3.0", the minor isn't defined. I suppose it's OK to do:
```suggestion
	local MINOR="0"
```